shared/src/store/action-reducers/simulation.ts

```diff
@@ -66,19 +71,19 @@ export namespace SimulationActionReducers {
                 const treatPatientsBehaviorState = behaviorStates.find(
                     (behaviorState) => behaviorState.id === behaviorStateId
                 ) as Mutable<TreatPatientsBehaviorState>;
-                if (unknown >= 0) {
+                if (unknown) {
```

Comment:
This way the values cannot be set to `0`.

Reply:
I added a better check